libs/driver/guides/magento.md

```diff
@@ -0,0 +1,66 @@
+
+# Magento/MageOS
+
+The `@daffodil/driver/magento` package configures the basic integration between your storefront and Magento 2/MageOS ecommerce platforms. 
+
+It provides the fundamental configurations that other Daffodil packages will use to interact with the relevant platforms. 
+
+Under the hood, this packages configures the Apollo GraphQl client to optimally query the Magento GraphQl api.
+
+## Features
+
+- GraphQl Caching via conversion of relevant requests to Varnish cacheable GET requests.
```

Comment:
```
It provides fundamental configurations that other Daffodil packages use to interact with these platforms. Under the hood, this package configures the Apollo GraphQL client to query the Magento GraphQL API.

- **GraphQL caching**: Converts relevant requests to Varnish-cacheable GET requests
  - Configurable by supplemental packages like `@daffodil/product` via `provideDaffMagentoCacheableOperation`
- **Apollo cache configuration**: Predefined `TypePolicies` and `PossibleTypes` for Apollo's `InMemoryCache`
- **Cache management**: Platform-specific cache clearing services for app state management and security via `DaffDriverHttpClientCacheServiceInterface`
```

Reply:
No, keep `## Features`.

I want `It provides fundamental configurations that other Daffodil packages use to interact with these platforms. Under the hood, this package configures the Apollo GraphQL client to query the Magento GraphQL API.` to be part of features instead of the main description.